doc/api/http2.md

```diff
@@ -3127,23 +3127,18 @@ added: v8.4.0
 
 * {string}
 
-Request URL string. This contains only the URL that is present in the actual
+Request target resource string. This contains the string from the actual
```

Comment:
“target resource string” sounds quite abstract – maybe just say it’s a relative URL?

Reply:
Yeah, my thinking here is misguided. I'm going to close this.